content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -661,6 +661,76 @@ of single-resource API requests, then aggregates the responses if needed.
 By contrast, the Kubernetes API verbs **list** and **watch** allow getting multiple
 resources, and **deletecollection** allows deleting multiple resources.
 
+## Field validation
+
+  {{< feature-state for_k8s_version="v1.25" state="beta" >}}
+
+Historically, in certain situations, the API server drops fields that it does not recognize
+from an object it receives.
```

Comment:
nit: AIUI  the message from the client can include any resource body; it doesn't have to be an object.

Reply:
Yea, I don't think there's anything here that is saying messages are exclusively objects, just that objects received from the clients will have their unknown fields dropped by the api server